executor/oomtest/oom_test.go

```diff
@@ -53,6 +54,9 @@ func TestMemTracker4UpdateExec(t *testing.T) {
 	log.SetLevel(zap.InfoLevel)
 
 	oom.SetTracker("")
+	oom.ClearMessageFilter()
+	oom.AddMessageFilter("expensive_query during bootstrap phase")
+	oom.AddMessageFilter("schemaLeaseChecker is not set for this transaction")
 
 	tk.MustExec("insert into t_MemTracker4UpdateExec values (1,1,1), (2,2,2), (3,3,3)")
 	require.Equal(t, "schemaLeaseChecker is not set for this transaction", oom.GetTracker())
```

Comment:
Line 59 add `schemaLeaseChecker is not set for this transaction` filter, is it expected?
Because line 62 check this error msg

Reply:
Yes， It means that it wants to get this log.